properties.py

```diff
@@ -390,7 +390,7 @@ def reset_ll_object_index(self, context):
 
     dark_falloff = FloatProperty(
         name="Dark Falloff",
-        description="Adds a small boost to pixel values in order to improve adaptive sampling",
+        description="Deprioritizes adaptive sampling in dark areas.  This can potentially reduce render times.",
```

Comment:
Ok ONE more edit to this... 
"Deprioritizes adaptive sampling in dark areas.  Raising this can potentially reduce render times but may increase noise in dark areas."